packages/forma-36-react-components/src/components/Card/CardLoading/__snapshots__/CardLoading.test.js.snap

```diff
@@ -412,9 +410,7 @@ exports[`renders the component with an additional class name 1`] = `
       timeout={100}
       unmountOnExit={true}
     >
-      <React.Fragment>
-        CardLoading
-      </React.Fragment>
+      CardLoading
```

Comment:
Did we throw out the Fragment or is this just the way the snapshot is rendered now?

Reply:
It seems to be just the way fragments are rendered now in Jest